WordPress/src/main/java/org/wordpress/android/modules/WordPressGlideModule.kt

```diff
@@ -22,6 +24,7 @@ import javax.inject.Named
 @GlideModule
 class WordPressGlideModule : AppGlideModule() {
     @Inject @field:Named("custom-ssl") lateinit var requestQueue: RequestQueue
+    @Inject @field:Named("noRedirects") lateinit var noRedirectsRequestQueue: RequestQueue
```

Comment:
NP: I think we can remove `field` here:

```suggestion
    @Inject @Named("custom-ssl") lateinit var requestQueue: RequestQueue
    @Inject @Named("noRedirects") lateinit var noRedirectsRequestQueue: RequestQueue
```